dashboard/client/pages/index.tsx

```diff
@@ -12,20 +12,32 @@ interface IProps {
   tickets: any[]
 }
 
-const HomePage: NextStatelessComponent<IProps> = ({ spaces, tickets }) => (
-  <Layout spaces={spaces}>
-    {tickets.map(t => (
-      <Ticket key={t.title} ticket={t} />
-    ))}
-  </Layout>
-)
+const HomePage: NextStatelessComponent<IProps> = ({ spaces, tickets: initalTickets, query }) => {
+  const [tickets, setTickets] = useState(initalTickets)
+
+  useEffect(() => {
+    const fetchTickets = async () => {
+      const { data } = await getTickets(query.space)
```

Comment:
we should be makin sure object properties even exist:

```
if (query && query.space) {
  const { data } = await getTickets(query.space)

  if (data && data.tickets) {
    setTickets(data.tickets)
  }
}
```

Reply:
query always exists w/ nextjs, if there's no query it's just an empty object